functions/listOrders/source.js

```diff
@@ -0,0 +1,40 @@
+exports = function(zipcodes){
+
+    /*  
+        Accepts an array of strings represting zipcodes.
+        Returns a JSON string representing an array of orders for those zipcode,
+            ordered by zipcode.
+        
+        To test this function in the Stitch UI:
+            Navigate to the function.
+            At the bottom, select console and pass in an array of zipcodes to the function call.
+            Then hit run.
+    */
+  
+    const atlas = context.services.get('mongodb-atlas');
+    
+    // Set placeholder.
+    let zips = ["1234", "2345", "3456"]
```

Comment:
Is there a reason why we need this place holder? I think we could just return an error if the parameter `zipcodes` is empty or invalid.

Reply:
Sure, I'll update.